python/orca/src/bigdl/orca/learn/pytorch/pytorch_pyspark_estimator.py

```diff
@@ -165,6 +167,12 @@ def __init__(
 
         self.state_dict = self.driver_runner.get_state_dict()
 
+    def create_tcpstore_server(self):
+        import torch.distributed as dist
+        from torch._C._distributed_c10d import _DEFAULT_PG_TIMEOUT
```

Comment:
We should avoid directly using torch private constants, since it may change in different torch versions. 

Firstly, ` _DEFAULT_PG_TIMEOUT` is only introduced after `1.9.0`. Being compatible with some lower versions might be good for user experience. 
Secondly,  it could introduce bugs, which might be very hard to locate, as torch evolves.

Reply:
Shall we just change it to 
```python
from dist.constants import default_pg_timeout
```

Reply:
> Shall we just change it to
> 
> ```python
> from dist.constants import default_pg_timeout
> ```

Maybe just use the default value `timeout: datetime.timedelta = datetime.timedelta(seconds=300)` ? Since the 30 minutes of `dafault_pg_timeout` may too many for creating server store on client node.

Reply:
> Maybe just use the default value timeout: datetime.timedelta = datetime.timedelta(seconds=300) ? Since the 30 minutes of dafault_pg_timeout may too many for creating server store on client node.

Pytorch DDP is built on the c10d library. And pytorch DDP uses a default timeout of `dafault_pg_timeout` to create TCPStore in `dist.init_process_group` ([code](https://github.com/pytorch/pytorch/blob/b4ed13ea0ff091328c6d0dfdf5d751d4280fb67f/torch/distributed/distributed_c10d.py#L488-L491)).

So I guess the default timeout of Pytorch DDP, instead of the default timeout of TCPStore in c10d library, should be more appropriate for our distributed training based on DDP.